### PR TITLE
Adtangir/gdap documentation

### PR DESCRIPTION
**Pull request by adtangir**

### CODE REVIEW

File: api-reference/beta/resources/delegatedadminrelationshiprequest.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "delegatedAdminRelationshipRequest resource type"
+description: "Represents a request specific to a delegated admin relationship between a partner and customer."
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/delegatedadminrelationshiprequest.md:3]: This description isn't completely clear when I look at the values of the two enums

File: api-reference/beta/resources/delegatedadminrelationshiprequest.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+doc_type: resourcePageType
+---
+
+# delegatedAdminRelationshipRequest resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents a request specific to a delegated admin relationship between a partner and customer.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/delegatedadminrelationshiprequest.md:16]: Please also make the change here

File: api-reference/beta/resources/delegatedadminrelationshiprequest.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+|[Delete delegatedAdminRelationshipRequest](../api/delegatedadminrelationshiprequest-delete.md)|None|Deletes a [delegatedAdminRelationshipRequest](../resources/delegatedadminrelationshiprequest.md) object.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|action|String|The action to be performed on the delegated admin relationship.|
+|createdDateTime|DateTimeOffset|The date and time (ISO 8601) at which the relationship request was created in UTC. This is set by the system and cannot be set by the caller.|
+|id|String|The unique identifier of the relationship request. This is set by the system and cannot be set by the caller.|
+|lastModifiedDateTime|DateTimeOffset|The date and time (ISO 8601) at which this relationship request was last modified in UTC. This is set by the system and cannot be set by the caller.|
+|status|String|The status of the request.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/delegatedadminrelationshiprequest.md:34]: status and action are enums. Please copy over the complete list of members from the doc stub.
```suggestion
|action|String|The action to be performed on the delegated admin relationship.|
|createdDateTime|DateTimeOffset|The date and time in ISO 8601 when the relationship request was created and is always in UTC. Read-only.|
|id|String|The unique identifier of the relationship request. Read-only.|
|lastModifiedDateTime|DateTimeOffset|The date and time in ISO 8601 when this relationship request was last modified and is always in UTC. Read-only.|
|status|String|The status of the request.|
```

File: api-reference/beta/api/delegatedadminrelationshiprequest-list.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-list.md:1]: File name is _delegatedadminrelationship-list-requests.md_

File: api-reference/beta/api/delegatedadminrelationshiprequest-list.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+title: "List delegatedAdminRelationshipRequests"
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-list.md:2]:
```suggestion
title: "List requests"
```

File: api-reference/beta/api/delegatedadminrelationshiprequest-list.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+description: "Get a list of the delegatedAdminRelationshipRequest objects and their properties."
+author: "smrtsec"
+ms.localizationpriority: medium
+ms.prod: "gdap"
+doc_type: apiPageType
+---
+
+# List delegatedAdminRelationshipRequests
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-list.md:10]:
```suggestion
# List requests
```

File: api-reference/beta/api/delegatedadminrelationshiprequest-list.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-list.md:37]: ...

File: api-reference/beta/api/delegatedadminrelationshiprequest-list.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "list_delegatedadminrelationshiprequest"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-list.md:60]: Real ID please

File: api-reference/beta/api/delegatedadminrelationshiprequest-list.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "Collection(microsoft.partner.customerServiceAdministration.delegatedAdminRelationshipRequest)"
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-list.md:76]: Real payload please

File: api-reference/beta/api/delegatedadminrelationship-post-requests.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
+---
+title: "Create delegatedAdminRelationshipRequest"
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationship-post-requests.md:2]:
```suggestion
title: "Create requests"
```

File: api-reference/beta/api/delegatedadminrelationship-post-requests.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+description: "Create a new delegatedAdminRelationshipRequest object."
+author: "smrtsec"
+ms.localizationpriority: medium
+ms.prod: "gdap"
+doc_type: apiPageType
+---
+
+# Create delegatedAdminRelationshipRequest
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationship-post-requests.md:10]:
```suggestion
# Create requests
```

File: api-reference/beta/api/delegatedadminrelationship-post-requests.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+``` http
+POST https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests
+Content-Type: application/json
+Content-length: 152
+
+{
+  "@odata.type": "#microsoft.partner.customerServiceAdministration.delegatedAdminRelationshipRequest",
+  "action": "String",
+  "status": "String"
+}
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationship-post-requests.md:73]: Real ID please
```suggestion
POST https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests
Content-Type: application/json

{
  "@odata.type": "#microsoft.partner.customerServiceAdministration.delegatedAdminRelationshipRequest",
  "action": "String",
  "status": "String"
}
```

File: api-reference/beta/api/delegatedadminrelationship-post-requests.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "microsoft.partner.customerServiceAdministration.delegatedAdminRelationshipRequest"
+}
+-->
+``` http
+HTTP/1.1 201 Created
+Content-Type: application/json
+
+{
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationship-post-requests.md:89]: Real payload please

File: api-reference/beta/api/delegatedadminrelationshiprequest-delete.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
+---
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-delete.md:1]: API.md claims Delete isn't supported.

File: api-reference/beta/api/delegatedadminrelationshiprequest-delete.md [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "delete_delegatedadminrelationshiprequest"
+}
+-->
+``` http
+DELETE https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests/{delegatedAdminRelationshipRequestId}
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-delete.md:57]: Real ID please

File: api-reference/beta/api/delegatedadminrelationshiprequest-delete.md [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+```
+
+
+### Response
+>**Note:** The response object shown here might be shortened for readability.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-delete.md:62]:
```suggestion
```

File: api-reference/beta/api/delegatedadminrelationshiprequest-get.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests/{delegatedAdminRelationshipRequestId}
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-get.md:37]: ..

File: api-reference/beta/api/delegatedadminrelationshiprequest-get.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "get_delegatedadminrelationshiprequest"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests/{delegatedAdminRelationshipRequestId}
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-get.md:60]: Real ID please

File: api-reference/beta/api/delegatedadminrelationshiprequest-get.md [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+{
+  "value": {
+    "@odata.type": "#microsoft.partner.customerServiceAdministration.delegatedAdminRelationshipRequest",
+    "id": "5a6666c9-7282-0a41-67aa-25a5a3fbf339",
+    "action": "String",
+    "status": "String",
+    "createdDateTime": "String (timestamp)",
+    "lastModifiedDateTime": "String (timestamp)"
+  }
+}
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-get.md:85]: Real payload please

File: api-reference/beta/api/delegatedadminrelationshiprequest-update.md [hunk 20]
```diff
@@ -0,0 +1,100 @@
...
+[!INCLUDE [table-intro](../../includes/update-property-table-intro.md)]
+
+
+|Property|Type|Description|
+|:---|:---|:---|
+|action|String|The action to be performed on the delegated admin relationship. Required.|
+|status|String|The status of the request. Optional.|
+|createdDateTime|DateTimeOffset|The date and time (ISO 8601) at which the relationship request was created in UTC. This is set by the system and cannot be set by the caller. Optional.|
+|lastModifiedDateTime|DateTimeOffset|The date and time (ISO 8601) at which this relationship request was last modified in UTC. This is set by the system and cannot be set by the caller. Optional.|
+
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-update.md:52]: Align these with the changes you make to the resource doc

File: api-reference/beta/api/delegatedadminrelationshiprequest-update.md [hunk 21]
```diff
@@ -0,0 +1,100 @@
...
+``` http
+PATCH https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests/{delegatedAdminRelationshipRequestId}
+Content-Type: application/json
+Content-length: 120
+
+{
+  "@odata.type": "#microsoft.graph.delegatedAdminRelationshipRequest",
+  "action": "String",
+  "status": "String"
+}
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-update.md:76]: Real IDs
Real request body
```suggestion
PATCH https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests/{delegatedAdminRelationshipRequestId}
Content-Type: application/json

{
  "@odata.type": "#microsoft.graph.delegatedAdminRelationshipRequest",
  "action": "String",
  "status": "String"
}
```

File: api-reference/beta/api/delegatedadminrelationshiprequest-update.md [hunk 22]
```diff
@@ -0,0 +1,100 @@
...
+<!-- {
+  "blockType": "response",
+  "truncated": true
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationshiprequest-update.md:91]: Real payload please

File: api-reference/beta/api/delegatedadminrelationship-list-requests.md [hunk 23]
```diff
@@ -0,0 +1,89 @@
...
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationship-list-requests.md:37]: ...

File: api-reference/beta/api/delegatedadminrelationship-list-requests.md [hunk 24]
```diff
@@ -0,0 +1,89 @@
...
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "list_delegatedadminrelationshiprequest"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/tenantRelationship/delegatedAdminRelationships/{delegatedAdminRelationshipId}/requests
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationship-list-requests.md:60]: Real ID please

File: api-reference/beta/api/delegatedadminrelationship-list-requests.md [hunk 25]
```diff
@@ -0,0 +1,89 @@
...
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "Collection(microsoft.partner.customerServiceAdministration.delegatedAdminRelationshipRequest)"
+}
+-->
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
```
Comment (FaithOmbongi) [inline api-reference/beta/api/delegatedadminrelationship-list-requests.md:76]: Real payload please

File: api-reference/beta/resources/delegatedadmincustomer.md [hunk 26]
```diff
@@ -0,0 +1,54 @@
...
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|displayName|String|The display name of the delegated admin customer organization. This is set by the system and cannot be set by the caller.|
+|id|String|The unique identifier of the delegated admin customer. This is set by the system and cannot be set by the caller.|
+|tenantId|String|The tenant ID of the delegated admin customer. Must be a valid tenant ID as in AAD.|
+
+## Relationships
+None.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/delegatedadmincustomer.md:35]: Missing **serviceManagementDetails** relationship

File: api-reference/beta/resources/delegatedadminservicemanagementdetail.md [hunk 27]
```diff
@@ -0,0 +1,43 @@
+---
+title: "delegatedAdminServiceManagementDetail resource type"
+description: "Contains the management details of a Delegated Admin service."
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/delegatedadminservicemanagementdetail.md:3]: Let's try to make the description clearer. How does this relate to the main delegatedAdminCustomer API?

File: api-reference/beta/resources/delegatedadminservicemanagementdetail.md [hunk 28]
```diff
@@ -0,0 +1,43 @@
...
+doc_type: resourcePageType
+---
+
+# delegatedAdminServiceManagementDetail resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Contains the management details of a Delegated Admin service.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/delegatedadminservicemanagementdetail.md:16]: Same changes you apply in the metadata, you'll apply here.

File: api-reference/beta/resources/delegatedadminservicemanagementdetail.md [hunk 29]
```diff
@@ -0,0 +1,43 @@
...
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|serviceId|String|The service ID of a managed service.|
+|serviceManagementUrl|String|The service name of a managed service.|
+|serviceName|String|The URL of the management portal for the managed service.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/delegatedadminservicemanagementdetail.md:23]: "Managed service" gives the vibes of a service provider. I don't think that's the message you've sent so far about the Partner-Customer relationship. I could be wrong.

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 30]
```diff
@@ -0,0 +1,166 @@
+{
+  "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "EnumType",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:7]:
```suggestion
          "ApiChange": "Enum type",
```

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 31]
```diff
@@ -0,0 +1,166 @@
...
+          "ChangedApiName": "delegatedAdminAccessAssignmentStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminAccessAssignmentStatus** enumeration type.",
+          "Target": "delegatedAdminAccessAssignmentStatus"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "EnumType",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:15]:
```suggestion
          "ApiChange": "Enum type",
```

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 32]
```diff
@@ -0,0 +1,166 @@
...
+          "ChangedApiName": "delegatedAdminAccessContainerType",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminAccessContainerType** enumeration type.",
+          "Target": "delegatedAdminAccessContainerType"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "EnumType",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:23]:
```suggestion
          "ApiChange": "Enum type",
```

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 33]
```diff
@@ -0,0 +1,166 @@
...
+          "ChangedApiName": "delegatedAdminRelationshipOperationStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipOperationStatus** enumeration type.",
+          "Target": "delegatedAdminRelationshipOperationStatus"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "EnumType",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:31]:
```suggestion
          "ApiChange": "Enum type",
```

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 34]
```diff
@@ -0,0 +1,166 @@
...
+          "ChangedApiName": "delegatedAdminRelationshipOperationType",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipOperationType** enumeration type.",
+          "Target": "delegatedAdminRelationshipOperationType"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "EnumType",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:39]:
```suggestion
          "ApiChange": "Enum type",
```

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 35]
```diff
@@ -0,0 +1,166 @@
...
+          "ChangedApiName": "delegatedAdminRelationshipRequestAction",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipRequestAction** enumeration type.",
+          "Target": "delegatedAdminRelationshipRequestAction"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "EnumType",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:47]:
```suggestion
          "ApiChange": "Enum type",
```

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 36]
```diff
@@ -0,0 +1,166 @@
...
+          "ChangedApiName": "delegatedAdminRelationshipRequestStatus",
+          "ChangeType": "Addition",
+          "Description": "Added the **delegatedAdminRelationshipRequestStatus** enumeration type.",
+          "Target": "delegatedAdminRelationshipRequestStatus"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "EnumType",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:55]:
```suggestion
          "ApiChange": "Enum type",
```

File: changelog/Microsoft.Partner.CustomerSvcAdmin.json [hunk 37]
```diff
@@ -0,0 +1,166 @@
...
+          "Description": "Added the [tenantRelationship](https://docs.microsoft.com/en-us/graph/api/resources/tenantRelationship?view=graph-rest-beta) resource",
+          "Target": "tenantRelationship"
+        }
+      ],
+      "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2022-02-16T23:33:05.8733913Z",
+      "WorkloadArea": "Microsoft.Partner.CustomerSvcAdmin",
+      "SubArea": ""
```
Comment (FaithOmbongi) [inline changelog/Microsoft.Partner.CustomerSvcAdmin.json:163]:
```suggestion
      "WorkloadArea": "Microsoft.Partner.CustomerSvcAdmin",
      "SubArea": ""
```
Let's use these values as per the internal thread. This API seems to be about directory management only in a partner-customer r/ship
```suggestion
      "WorkloadArea": "Identity and access",
      "SubArea": "Directory management"
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @adtangir for your work on this PR. The PR was huge but you did an excellent job trying to get all required docs into the PR.

I've left suggestions for changes/where to make things clearer etc. In addition, please address the following:
- GLOBAL: The assigned author of the docs should be a user's GitHub alias, not a team's internal mailing alias as per the [Get started steps](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub). I see **amharris1331** has the necessary memberships  - will she own the feature, docs, and be available for support?
- Please document the **delegatedAdminRelationships** + **delegatedAdminCustomers** relationships in the tenantRelationship resource doc.
- GLOBAL: I think the **directory-management** ms.prod value is suited for these APIs. From what I've understood, they are mostly about delegated administrative role assignments/management between a Partner and a customer tenant.
- We'll need an Overview article to explain the whole crux of these new APIs - how they relate to the Partner Center, VAR, etc. Building blocks. Benefits. Example scenarios. Licensing requirements etc.
- I see you're using the namespace `microsoft.partner.customerServiceAdministration.`. This is also the cause of 3/4 of the validation marked "Unable to locate a definition for resource type: microsoft.partner.customerServiceAdministration.****" I think [ALL Microsoft Graph APIs](https://docs.microsoft.com/en-us/graph/use-the-api#odata-namespace) are defined in the `microsoft.graph` namespace, even though some might be defined in sub-namespaces. If confirmed microsoft.graph namespace is correct, please change all occurrences of `microsoft.partner.customerServiceAdministration.*` to `microsoft.graph.*`
- GLOBAL: All examples in the API articles should be real examples with real data instead of placeholder data types. Customers report to find these more helpful. Please run your queries in a demo tenant to get the values. 
- I see you've introduced two new permissions, please document them in the [permissions reference](docs.microsoft.com/en-us/graph/permissions-reference) doc.
- I see some operations support optional and required headers. Let's document all of them.
- Your API.md claims that delegatedAdminCustomer resource supports only LIST and GET. I see this note: "Not supported; will be supported in future for internal applications only.". Please confirm and make the changes/remove files or entries where applicable.
- delegatedadminoperationmd should be **delegatedadminrelationshipoperation.md** because I see the resource name changed between the API.md approval and the schema release
- The API.md says that CREATE, DELETE, UPDATE operations aren't supported by delegatedAdminRelationshipOperation API. Please confirm.
- Missing APIs for serviceManagementDetails. I see the URL pattern `GET /delegatedAdminCustomers/{ID}/serviceManagementDetails` in the API.md

File: api-reference/beta/resources/delegatedadminrelationship.md [hunk 38]
```diff
@@ -0,0 +1,86 @@
...
+|id|String|The unique identifier of the relationship. This is set by the system and cannot be set by the caller.|
+|lastModifiedDateTime|DateTimeOffset|The date and time (ISO 8601) at which this relationship was last modified in UTC. This is set by the system and cannot be set by the caller.|
+|displayName|String|The display name of the relationship. This is primarily meant for ease of identification. This is set by the partner and cannot be changed by the customer, and cannot be changed by the caller once the relationship is in the "approvalPending" status or beyond. Must be unique across all relationships from the partner for the customer.|
+|partner|[delegatedAdminRelationshipParticipant](../resources/delegatedadminrelationshipparticipant.md)|The information of the partner of the relationship. This is set by the partner and cannot be changed by the customer.|
+|status|delegatedAdminRelationshipStatus|The status of the relationship.The possible values are: `activating`, `active`, `approvalPending`, `approved`, `created`, `expired`, `expiring`, `terminated`, `terminating`, `terminationRequested`, `unknownFutureValue`.|
```
[the remark this answers is not in the document]
Comment (adtangir) [reply api-reference/beta/resources/delegatedadminrelationship.md:46]: The schema for some of the fields above has changed and we have updated the field descriptions accordingly. Please give these a second pass

File: api-reference/beta/resources/delegatedadminrelationship.md [hunk 39]
```diff
@@ -0,0 +1,86 @@
+---
+title: "delegatedAdminRelationship resource type"
+description: "Represents the partner view of a delegated admin relationship between a partner and customer."
+author: "smrtsec"
```
[the remark this answers is not in the document]
Comment (adtangir) [reply api-reference/beta/resources/delegatedadminrelationship.md:4]: I'll check with the team and update the this with a better description

File: api-reference/beta/resources/delegatedadminrelationship.md [hunk 40]
```diff
@@ -0,0 +1,86 @@
...
+ms.localizationpriority: medium
+ms.prod: "gdap"
```
[the remark this answers is not in the document]
Comment (adtangir) [reply api-reference/beta/resources/delegatedadminrelationship.md:6]: Updated the ms.prod value to directory-management

**Comment by FaithOmbongi** [comment]
The changes are live. Removing **changes requested** label to avoid unactionable notifications.

**Pull request merged**